### PR TITLE
Enable async_io for squad.

### PR DESCRIPTION
**Pull request by trentlo**
This reduces Horovod overhead when XLA is enabled.

~7-9% speedup is observed for BERT Squad fp32 and ~3-4% speedup for fp16.

### CODE REVIEW

File: TensorFlow/LanguageModeling/BERT/run_squad.py [hunk 1]
```diff
@@ -940,6 +940,9 @@ def main(_):
   else:
     os.environ["TF_XLA_FLAGS"] = "--tf_xla_enable_lazy_compilation=false"
 
+  # Enable async_io to speed up multi-gpu training with XLA and Horovod.
+  os.environ["TF_XLA_FLAGS"] += "--tf_xla_async_io_level=1"
```
Comment (bas-aarts) [inline TensorFlow/LanguageModeling/BERT/run_squad.py:944]: no " " separator?
same on line 939

Comment (trentlo) [reply TensorFlow/LanguageModeling/BERT/run_squad.py:944]: You are right. I was misled by line 939. I wonder why it worked?

Comment (trentlo) [reply TensorFlow/LanguageModeling/BERT/run_squad.py]: Did some tests. So, the line 939 never worked...

I updated the PR. Thanks @bas-aarts for the catch.

Comment (bas-aarts) [reply TensorFlow/LanguageModeling/BERT/run_squad.py]: looks good

**Pull request merged**